app/screens/edit_post/edit_post.js

```diff
@@ -279,6 +279,8 @@ export default class EditPost extends PureComponent {
                         value={message}
                         nestedScrollEnabled={true}
                         onVisible={this.onAutocompleteVisible}
+                        rootId={this.props.post.root_id}
```

Comment:
Why do we want slash command autocomplete in edit post ?

Reply:
It was already there. When you edit post you can use command autocomplete. I thought it's a feature, isn't it? :D

Reply:
lol... it seems weird to me and I doubt it works,.. maybe create another ticket to remove slash commands from edit post ?

Reply:
It's not executing for sure, but you can type and autocomplete commands. Should I push those changes here as well?

Reply:
If you can do it, that'll be great.. if not a new ticket is fine.

Reply:
@enahum FYI removed autocomplete on edit_post, as well as on edit_channel_info